building/trusted_applications.rst

```diff
@@ -373,35 +373,39 @@ third step the binary and its signature are stitched together into the full TA.
 Offline Signing of TAs
 ======================
 
-The TA dev kit does sign an application as last step of the linking process. For
-example, the file ``ta/arch/arm/link.mk`` in the :ref:`optee_os` source tree
-contains the statement
+There are two types of TAs that can be signed offline: the so-called `in-tree` TAs, which come
+with the OPTEE OS (e.g. the ``pkcs11`` TA) and any external TA coming from the user. Offline signing
```

Comment:
please replace "e.g." with "for example".

Reply:
The sentence "Offline signing..." is a bit strange. Does offline signing really run? Perhaps it can also be split in two?

Reply:
Yep you are right. What about "Offline Signing is carried out"? My point here is that even the `make all` generates the `.elf` and `.ta` files, the offline signing itself signs the `.stripped.elf` files.

Reply:
That's better.

Reply:
Fixed!